MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -152,6 +158,7 @@ object VisionManager : BaseVisionManager {
         this.application = application
         this.mapboxToken = mapboxToken
         PreferencesManager.appContext = application
+        performanceProvider = PerformanceProvider.Impl(application).addTo(attachedModules)
```

Comment:
Can't recall why we need to init it in `init`, would it be better to do in `onCreate`?

Reply:
didn't get your point in previous comment. I'm agree